hamilton/plugins/ibis_extensions.py

```diff
@@ -0,0 +1,75 @@
+from typing import Type
+
+from hamilton import registry
+
+try:
+    import ibis
+    import ibis.expr.types as ir
+except ImportError:
+    raise NotImplementedError("Ibis is not installed.")
+
+from hamilton.data_quality import base, default_validators
+
+DATAFRAME_TYPE = ir.Table
+COLUMN_TYPE = ir.Column
+
+
+def view_expression(expression: ir.Expr, **kwargs):
```

Comment:
if we want people to use this -- then this should be under `h_ibis.py`.

Reply:
Yeah to add to what @skrawcz said there are two extensions you want:
1. `h_ibis` -- this is anything that gets imported
2. `ibis_extensions` -- this is stuff that gets auto-registered